internal/proto/opamp.proto

```diff
@@ -660,32 +668,33 @@ message PackageStatus {
     // Server offers a different version, but the Agent fails to install that version.
     bytes server_offered_hash = 5;
 
-    // The status of this package.
-    enum Status {
-        // Package is successfully installed by the Agent.
-        // The error_message field MUST NOT be set.
-        Installed = 0;
-
-        // Installation of this package has not yet started.
-        InstallPending = 1;
-
-        // Agent is currently downloading and installing the package.
-        // server_offered_hash field MUST be set to indicate the version that the
-        // Agent is installing. The error_message field MUST NOT be set.
-        Installing = 2;
-
-        // Agent tried to install the package but installation failed.
-        // server_offered_hash field MUST be set to indicate the version that the Agent
-        // tried to install. The error_message may also contain more details about
-        // the failure.
-        InstallFailed = 3;
-    }
-    Status status = 6;
+    PackageStatusEnum status = 6;
 
     // Error message if the status is erroneous.
     string error_message = 7;
 }
 
+// The status of this package.
+enum PackageStatusEnum {
```

Comment:
nit: Should we call this PackageStatuses to match other similar enum names? This is the only one with an Enum suffix.

Reply:
Unfortunately I can't because we already have a message named `PackageStatuses`. This was a forced deviation.